src/features/channels/components/channel-list/search-channel/styles.css

```diff
@@ -0,0 +1,12 @@
+.MuiInputLabel-root {
```

Comment:
I'd suggest to not use plain css files, and keep using the Material UI styling mechanism as others. If you need to apply global css rules, you can use styleOverrides prop in the theme file